src/core/instance/render-helpers/bind-object-props.js

```diff
@@ -1,8 +1,11 @@
 /* @flow */
 
 import config from 'core/config'
+import { makeMap } from 'shared/util'
 import { isObject, warn, toObject } from 'core/util/index'
 
+const isSpecialAttributes = makeMap('key,ref,slot,is')
```

Comment:
@yyx990803 

sorry,  It was too late yesterday, I will finish after work today

Do not we need to consider `is`?